src/integrationTest/java/uk/gov/hmcts/darts/event/controller/EventsControllerPostEventsTest.java

```diff
@@ -35,7 +35,6 @@
 @Transactional
```

Comment:
Can't remember if we discussed this particular instance of Transaction already, but if not can we avoid this on an int test?